tests/30rooms/12thirdpartyinvite.pl

```diff
@@ -181,29 +175,25 @@ sub can_invite_unbound_3pid
    };
 
 sub invite_should_fail {
-   my ( $inviter, $invitee, $hs_base_url, $bind_sub ) = @_;
+   my ( $inviter, $invitee, $hs_base_url, $id_server, $bind_sub ) = @_;
```

Comment:
Unrelated: $hs_base_url is unused; we should stop passing it at some point. I can pick this up when this lands.